Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -344,38 +344,47 @@ bool VolumeVerifier::IsDebugSigned() const
 
 bool VolumeVerifier::ShouldHaveChannelPartition() const
 {
-  const std::unordered_set<std::string> channel_discs{
-      "RFNE01", "RFNJ01", "RFNK01", "RFNP01", "RFNW01", "RFPE01", "RFPJ01", "RFPK01", "RFPP01",
-      "RFPW01", "RGWE41", "RGWJ41", "RGWP41", "RGWX41", "RMCE01", "RMCJ01", "RMCK01", "RMCP01",
+  static constexpr std::array<std::string_view, 18> channel_discs = {
+      "RFNE01", "RFNJ01", "RFNK01", "RFNP01", "RFNW01",
+      "RFPE01", "RFPJ01", "RFPK01", "RFPP01", "RFPW01",
+      "RGWE41", "RGWJ41", "RGWP41", "RGWX41",
+      "RMCE01", "RMCJ01", "RMCK01", "RMCP01",
   };
 
-  return channel_discs.find(m_volume.GetGameID()) != channel_discs.end();
+  return std::binary_search(channel_discs.cbegin(), channel_discs.cend(),
+    m_volume.GetGameID().c_str(),
+    [](const std::string_view a, const std::string_view b) { return a < b; });
```

Comment:
Is this predicate really needed? Looks like something which is already defined by STL.

Reply:
I checked and you're right, the predicate isn't needed; the default should work.